### PR TITLE
Fix desc labels

### PR DESCRIPTION
**Pull request by luisremis**
Check just the last two commit. 
This has to be rebased to develop before merging, and only after the Python test refactor is merged.

### CODE REVIEW

**Comment by luisremis** [comment]
This has been rebased to develop, ready for merge. @vishakha041

File: src/DescriptorsCommand.cc [hunk 1]
```diff
@@ -269,6 +269,10 @@ int AddDescriptor::construct_protobuf(
         label = cmd["label"].asString();
         props[VDMS_DESC_LABEL_PROP] = label;
     }
+    else {
+        label = "None";
+        props[VDMS_DESC_LABEL_PROP] = label;
```
Comment (vishakha041) [inline src/DescriptorsCommand.cc]: Could you pull this line out of the if else?

Comment (luisremis) [reply src/DescriptorsCommand.cc]: actually, I realized we have another mechanism to do that.

**Comment by luisremis** [comment]
lgtm!

File: src/DescriptorsCommand.cc [hunk 2]
```diff
@@ -264,11 +264,8 @@ int AddDescriptor::construct_protobuf(
 
     Json::Value props = get_value<Json::Value>(cmd, "properties");
 
-    std::string label;
-    if (cmd.isMember("label")) {
-        label = cmd["label"].asString();
-        props[VDMS_DESC_LABEL_PROP] = label;
-    }
+    std::string label = get_value<std::string>(cmd, "label", "None");
+    props[VDMS_DESC_LABEL_PROP] = label;
```
Comment (vishakha041) [inline src/DescriptorsCommand.cc]: Much better :)

**Pull request merged**